### ISSUE TITLE
[Bug]: islington council 

### ISSUE THREAD

**Issue by BustaChewy** [mampfes/hacs_waste_collection_schedule#3040]
### I Have A Problem With:

A specific source

### What's Your Problem

I get this error message when adding the integration after selecting islington council and my unique address ref. 

Error msg "The source returned an invalid response: "'NoneType' object has no attribute 'text'". Please check the provided arguments and try again."

### Source (if relevant)

Islington council

### Relevant Configuration

```YAML
Using the lastest version of home assistant and all add ons
```

### Checklist Source Error

- [ ] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [ ] Checked that the website of your service provider is still working
- [ ] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by ReneNulschDE** [comment]
Please share your configuration.

**Comment by BustaChewy** [comment]
Pi 4GB running home assisitant os 13.2  core 2023.11.1, supervisor
2014.11.2.

Thanks

**Comment by ReneNulschDE** [comment]
I mean your config of WCS aka your UNIQUE_PROPERTY_REFERENCE_NUMBER or a number with the same problem.

**Comment by BustaChewy** [comment]
Thanks Rene, it's 5300078702.
🙌

**Comment by BustaChewy** [comment]
Hi Rene, I saw you updated the posting saying it needs the post code now.

I added the post code and still get the same error, apologises this is my
1st year using HA - am I missing something obvious to get this working?
Thanks Rene 🙌

**Comment by ReneNulschDE** [comment]
You need the next version of this component too (... not available yet)

**Comment by BustaChewy** [comment]
Ah - I'm jumping the gun, will hold tight. Thanks mate

**Comment by 5ila5** [comment]
Fixed in #3040,

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>

You will need to reconfigure your source as it now requires a postcode
